util/src/main/java/org/kaazing/gateway/util/InternalSystemProperty.java

```diff
@@ -129,6 +128,11 @@
     GATEWAY_IDENTIFIER
             ("org.kaazing.gateway.server.GATEWAY_IDENTIFIER", ""),
 
+    // Internal system property checking if a newer version of the Gateway is available
+    UPDATE_CHECK("org.kaazing.gateway.server.UPDATE_CHECK", "true"),
+
+    SERVICE_URL("org.kaazing.gateway.server.SERVICE_URL"),
```

Comment:
ok, please rename to `UPDATE_CHECK_SERVICE_URL` or `UPDATE_CHECK_URL`, service is an ambiguous term due to several different contexts it can be used in